homework3/semi-uber/Server/index.js

```diff
@@ -2,29 +2,25 @@ const express = require("express");
 const app = express();
 const mongoose = require("mongoose");
 const config = require("config");
+const registrationRouter = require("./api/routes/registration.router");
+const allowCrossDomain = require("./api/middlewars/allowCrossDomain");
 
 const writeLog = require("./api/middlewars/writeLog");
 const { port: serverPort } = config.get("webServer");
 const { protocol, host, port, name } = config.get("dataBase");
 const dbURL = `${protocol}://${host}:${port}/${name}`;
 
-
 mongoose.connect(dbURL, {
   useNewUrlParser: true
 });
 
-const User = mongoose.model(
-  "User",
-  mongoose.Schema({ username: String, password: String })
-);
-
 app.use(express.json());
-
+app.use(allowCrossDomain);
 app.use("../api", express.static("api"));
 
-app.get("/", writeLog, (req, res) => {
-  User.find({}).then(user => res.json(user));
-});
+app.get("/", writeLog);
```

Comment:
I am not sure you really need this route

Reply:
Need it to write log, when anyone will visit site, from the first step